docs/resources/aws_vpn_connections.md

```diff
@@ -0,0 +1,67 @@
+---
+title: About the aws_vpn_connections Resource
+platform: aws
+---
+
+# aws\_vpn\_connections
+
+Use the `aws_vpn_connections` InSpec audit resource to test properties of all AWS VPN Connections.
+
+## Syntax
+
+Ensure that one or more VPN Connections exist.
+
+    describe aws_vpn_connections do
+      it { should exist }
+    end
+
+## Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on VPN Connections](https://docs.aws.amazon.com/vpc/latest/userguide/vpn-connections.html).
+
+## Properties
+
+|Property           | Description|
+| ---               | --- |
+|vpn\_connection\_id    | This property provides all the IDs of the VPN Connections |
```

Comment:
Perhaps it is better to make this plural since it is an array of VPN connection ids.